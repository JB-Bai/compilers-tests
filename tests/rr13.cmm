float;
struct{
}
;
int;
struct{
}
XCMg2zie(){
	(IhtspFuv);
	return 0xAecad && 01225 || - 0x0A0a[68520][64.15E-1][! uYQTo4CjRCv.NAqZZgYL];
}
struct{
}
rVXZPNn1lY, P0nj8hpm7n[0411], Om1X3aN4H;

