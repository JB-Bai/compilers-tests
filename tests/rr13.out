Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: float
      SEMI
    ExtDefList (2)
      ExtDef (2)
        Specifier (2)
          StructSpecifier (2)
            STRUCT
            LC
            RC
        SEMI
      ExtDefList (5)
        ExtDef (5)
          Specifier (5)
            TYPE: int
          SEMI
        ExtDefList (6)
          ExtDef (6)
            Specifier (6)
              StructSpecifier (6)
                STRUCT
                LC
                RC
            FunDec (8)
              ID: XCMg2zie
              LP
              RP
            CompSt (8)
              LC
              StmtList (9)
                Stmt (9)
                  Exp (9)
                    LP
                    Exp (9)
                      ID: IhtspFuv
                    RP
                  SEMI
                StmtList (10)
                  Stmt (10)
                    RETURN
                    Exp (10)
                      Exp (10)
                        Exp (10)
                          INT: 715949
                        AND
                        Exp (10)
                          INT: 661
                      OR
                      Exp (10)
                        MINUS
                        Exp (10)
                          Exp (10)
                            Exp (10)
                              Exp (10)
                                INT: 2570
                              LB
                              Exp (10)
                                INT: 68520
                              RB
                            LB
                            Exp (10)
                              FLOAT: 6.415000
                            RB
                          LB
                          Exp (10)
                            NOT
                            Exp (10)
                              Exp (10)
                                ID: uYQTo4CjRCv
                              DOT
                              ID: NAqZZgYL
                          RB
                    SEMI
              RC
          ExtDefList (12)
            ExtDef (12)
              Specifier (12)
                StructSpecifier (12)
                  STRUCT
                  LC
                  RC
              ExtDecList (14)
                VarDec (14)
                  ID: rVXZPNn1lY
                COMMA
                ExtDecList (14)
                  VarDec (14)
                    VarDec (14)
                      ID: P0nj8hpm7n
                    LB
                    INT: 265
                    RB
                  COMMA
                  ExtDecList (14)
                    VarDec (14)
                      ID: Om1X3aN4H
              SEMI
